simulation/amaru-sim/src/simulator/args.rs

```diff
@@ -51,6 +51,10 @@ pub struct Args {
     /// Persist generated data and pure-stage traces even if the test passes.
     #[arg(long)]
     pub persist_on_success: bool,
+
+    /// Directory where test data must be persisted
+    #[arg(long, default_value = "../../target/tests")]
```

Comment:
hmm, not sure having a relative directory by default is a good idea, if only for security reasons. how about just `test-data` ?

Reply:
By default I'd like the output to go to `target` so we won't ever make the mistake of committing the files by accident. Now that I write this, the other option would be to put those file in a `test-data` directory and add that file to `.gitignore`.